eclass/qt5-build.eclass

```diff
@@ -67,10 +67,11 @@ QT5_MINOR_VERSION=$(ver_cut 2)
 readonly QT5_MINOR_VERSION
 
 case ${PV} in
-	5.??.9999)
-		# git stable branch
+	5.15.9999)
+		# KDE upstream for 5.15 patches
+		HOMEPAGE+="https://invent.kde.org/qt/qt/${QT5_MODULE}/-/tree/kde/5.15"
```

Comment:
I don't think HOMEPAGE can contain variables[1]. Maybe just point to https://invent.kde.org/qt/qt ?

[1] https://devmanual.gentoo.org/ebuild-writing/variables/index.html#ebuild-defined-variables

Reply:
Ah, right.  Completely forgot about that.

Changed as suggested.